java/client/src/main/java/glide/utils/ArrayTransformUtils.java

```diff
@@ -78,7 +79,7 @@ public static GlideString[] convertNestedArrayToKeyValueGlideStringArray(GlideSt
             }
         }
         return Arrays.stream(args)
-                .flatMap(entry -> Stream.of(entry[0], GlideString.gs(entry[1].toString())))
+                .flatMap(entry -> Stream.of(entry[0], entry[1]))
```

Comment:
Please fix docs for this function.  It mentions "of any type" twice in the javadoc.